bluetooth: Shell 'init' and 'advertise' commands rework (#11604)
zephyrproject-rtos/zephyr: doc/subsystems/shell/shell.rst

		return 0;
	}

> :cpp:func:`shell_fprintf` or shell print macros: :c:macro:`shell_print`,

Reviewer: ```suggestion
Function :cpp:func:`shell_fprintf` or the shell print macros: :c:macro:`shell_print`,
```

> :c:macro:`shell_info`, :c:macro:`shell_warn`, :c:macro:`shell_error` can only be

Reviewer: ```suggestion
:c:macro:`shell_info`, :c:macro:`shell_warn`, and :c:macro:`shell_error` can only be
```

> used from the command handler unless command context is forced to stay in

Reviewer: ```suggestion
used from the command handler, or if the command context is forced to stay in
```

> foreground by calling :cpp:func:`shell_command_enter` from within command

Reviewer: ```suggestion
the foreground by calling :cpp:func:`shell_command_enter` from within the command
```

> handler. In that case, shell stops reading input and writing to the output

Reviewer: ```suggestion
handler. In this latter case, the shell stops reading input and writing to the output
```

> (except for the logs), allowing user print from	any thread context.

Reviewer: ```suggestion
(except for the logs), allowing a user to print from any thread context.
```

> :cpp:func:`shell_command_exit` or CTRL+C terminates such 'foreground' command.

Reviewer: ```suggestion
Function :cpp:func:`shell_command_exit` or entering a :kbd:`CTRL+C` terminates a 'foreground' command.
```

> Foreground command implementation:

Reviewer: ```suggestion
Here is an example foreground command implementation:
```
